src/en/config-azure.md

```diff
@@ -1,123 +1,223 @@
-
-**Note:** Other than `location` the other key value defaults are recommended,
-but can be updated to your preference.
-
-**Note:** Ensure that `management-certificate-path` is set to use the .pem
-file, NOT the .cer file, doing so will result in an
-[out of memory error](https://bugs.launchpad.net/ubuntu/+source/juju-core/+bug/1250007).
-
-
-## Config Values
-
-Generate a new certificate for Juju usage (or use an existing one). Suggest to
-use a 'Common Name' with 'Juju' in it so it's obvious when viewed in the web UI
-later. On Ubuntu, run the following commands to generate a new certificate:
-
-```bash
-openssl req -x509 -nodes -days 3650 -newkey rsa:2048 -keyout azure.pem -out azure.pem
-openssl x509 -inform pem -in azure.pem -outform der -out azure.cer
-chmod 600 azure.pem
-```
-
-Log in to the Windows Azure console at
-[https://manage.windowsazure.com](https://manage.windowsazure.com). From there
-you can gather the following information:
-
-- **Settings** Left Navigation: get the `SUBSCRIPTION ID` of the account you
-  upload to. This will be used for the `management-subscription-id`
-- **Settings** Left Navigation: "upload" the MyCert.cer file above (if you
-  created one). This is the certificate you used for the
-  `management-certificate-path`
-- **Storage** Left Navigation:
-  - Click New (bottom left)
-  - Click Quick Create
-  - Add a name in url (for example `juju0useast0`). This is the value to be
-    used for `storage-account-name`.
-  - Select Location (for example: West US)
-  - Select Subscription
-  - Disable "Enable Geo-Replication" (not applicable)
-
-**Note:** You must create the storage account in the same region/location
-specified by the `location` key value. For example, if `location: West US` is
-set then `storage-account-name:` must also have a storage set up in `West US`.
-Failure to do so will result in a group affinity error.
-
-Edit `environments.yaml` according to the above configuration settings and save
-it to disk.
-
-
-## Using Availability Sets
-
-With Azure, each Cloud Service has zero or more Availability Sets within it and
-a Role can be assigned to at most one of them. As long as there are at least
-two Roles in the same Availability Set, then Azure will guarantee at least
-99.95% availability under the Azure Service Level Agreement (SLA).
-
-Juju creates a single Availability Set for each Cloud Service, and all Roles
-are added to it. Thus, all Juju-deployed services are, by default, covered by
-the Azure SLA.
-
-You can read more about
-[Azure Availability Sets](http://azure.microsoft.com/en-gb/documentation/articles/virtual-machines-manage-availability/).
+        storage-account-type: Standard_LRS
+        location: East US
+        subscription-id: f717c8c1-8e5e-4d38-be7f-ed1e1c879e1a
+        application-password: some_password
+        application-id: f6ab7cbd-5029-43ef-85e3-5c4442a00ba8
+        tenant-id: daff614b-725e-4b9a-bc57-7763017c1cfb
+```
```

Comment:
might be best to leave the actual yaml bits for azure in, eg:
   azure:
         type:azure
         ...
         ...